### ISSUE TITLE
 variable aspect ratios, resolutions, durations

### ISSUE THREAD

**Issue by SmileTAT** [PKU-YuanGroup/Open-Sora-Plan#56]
Is the implementation of variable aspect ratios, resolutions, durations different from that of NaViT? Are there any plans to implement the NaViT?

**Comment by LinB203** [comment]
Good idea, have you any plan to pull a request? This is a awesome todo. We add this.

**Comment by bhack** [comment]
We could also evaluate https://openreview.net/forum?id=uoRbMNoZ7w

**Comment by SkylerZheng** [comment]
What about supporting FiT for T2V?
